fairmq/DeviceRunner.cxx

```diff
@@ -33,26 +36,34 @@ auto DeviceRunner::Run() -> int
     fEvents.Emit<hooks::SetCustomCmdLineOptions>(*this);
     ////////////////////////
 
-    fPluginManager.ForEachPluginProgOptions([&](boost::program_options::options_description options){
-        fConfig.AddToCmdLineOptions(options);
-    });
+    fPluginManager.ForEachPluginProgOptions(
+        [&](boost::program_options::options_description options) {
+            fConfig.AddToCmdLineOptions(options);
+        });
     fConfig.AddToCmdLineOptions(fPluginManager.ProgramOptions());
 
     ////// CALL HOOK ///////
     fEvents.Emit<hooks::ModifyRawCmdLineArgs>(*this);
     ////////////////////////
 
-    if (fConfig.ParseAll(fRawCmdLineArgs, true))
-    {
+    if (fConfig.ParseAll(fRawCmdLineArgs, true)) {
         return 0;
     }
 
+    if (fPrintLogo) {
+        LOG(info) << std::endl
```

Comment:
Make this logo ultra fast by avoiding unnecessary flushes - replace `std::endl` with `'\n'`.

Reply:
Is there a fast and portable line break?